reaper/update.ps1

```diff
@@ -33,13 +33,19 @@ function global:au_GetLatest {
   # Get the actual download URL
   $download_base = "https://reaper.fm/"
 
-  $installer_32_regex = "reaper.*[^_64]-install.exe"
-  $installer_32_exe = $download_page.Links | ? href -match $installer_32_regex | Select-Object -First 1 -expand href
+  # Find 32-bit installer by looking for "Windows 32-bit" in the title
+  $installer_32_link = $download_page.Links | 
+  Where-Object { $_.title -match "Windows 32-bit" } | 
```

Comment:
I do not see how it's safer to look for a page name instead of the targeted installer file to download in the links. The former can change without notice, while the later is very unlikely to change.

Reply:
The current regex implementation was incorrectly matching the ARM build which is now included on the download page.  This is not matching a page name, it uses the title attribute of the href tag which seems to be more predictable.

Reply:
The <a> tag has only one required attribute, href. Relying on optional attribute content is not reliable. And I doubt chocolatey reviewers accept something else nowadays. Fixing the regexp to ignore arm installer binaries is easy while staying reliable.

Reply:
I do not any attempt at using the only one reliable attribute in `<a>` tag. What will happen when reaper webmaster will change the HTML code structure or the title because of a design revamp (which happens more often than the target binaries namespace)? The chocolatey package will break for no good reason.

Reply:
I think the only sensible way to handle that is a `.portable` package. I had to make substantive changes to make this compatible with the 7.35 release, and I think this is an improvement over that.

Reply:
Which is still lesser than the timeframe since the reaper binaries namespace have changed, still use an optional attribute in an HTML tag instead of a required one, which not reliable per design. 
And still revert a piece of code to a more fragile implementation instead of just fixing a regexp. 
Whatever, I've warned about the potential minor regression here and do not have time to do more ;-)